plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseTestHiveOnDataLake.java

```diff
@@ -66,7 +66,7 @@
 public abstract class BaseTestHiveOnDataLake
         extends AbstractTestQueryFramework
 {
-    private static final String HIVE_TEST_SCHEMA = "hive_datalake";
+    private static final String HIVE_TEST_SCHEMA = "Hive_Datalake";
```

Comment:
Add a comment that it is important that you use mixed case here

Reply:
Applied